src/wp-includes/block-template-utils.php

```diff
@@ -1468,6 +1468,37 @@ function inject_ignored_hooked_blocks_metadata_attributes( $post, $request ) {
 	$template = $request['id'] ? get_block_template( $request['id'], $post_type ) : null;
 	remove_filter( 'hooked_block_types', '__return_empty_array', 99999 );
 
+	if ( $template ) {
+		// TODO: Should maybe make this static. E.g. in the Templates Controller?
```

Comment:
Hmm, for my own benefit can you explain how you see this TODO being implemented a little more please? I'm not quite following.

Reply:
I mostly meant that it'd be nice if the mappings were (more) publicly available, since I ended up [duplicating](https://github.com/WordPress/wordpress-develop/pull/6254/files#diff-d5470ce0db8dafe9e9d34bc2dd1e9c5d3f67bdfd365427b91a0ab438b0382314R516-R524) them in the unit test. 

(Making them `static` was a poor choice of words on my part, where I conflated the means with the end.)

I realize now that a publicly available mappings array is probably not totally what we'd want, since there are [some fields](https://github.com/WordPress/wordpress-develop/pull/6254/files#diff-1d8e05d3ea69557ca8742330e62c1893a72d4ca4ff2ed4ccc5b94bcf5d20fed7R1491-R1499) for which there isn't a 1:1 mapping from our `$changes` object to `WP_Block_Template`, so it'd have to be a function at least.

In the long run, this might be something we could use inside of `_build_block_template_result_from_post` (it'd need some adjusting though to account for the differences between mapping from `WP_Post` -- rather than from our `$changes` `stdClass` -- to `WP_Block_Template`). This also means that it should _not_ be a static member of the Templates controller after all, as functions such as `_build_block_template_result_from_post` are clearly lower-level than the controller.

Reply:
I'll rephrase the TODO comment accordingly.